aemedge/blocks/content-list/content-list.css

```diff
@@ -1,103 +1 @@
-.content-list {
-  container: content-list / inline-size;
-
-  & .heading {
-    grid-column: 1 / -1;
-    margin: unset;
-  }
-
-  & ul {
-    display: grid;
-    grid-template-columns: repeat(auto-fill, minmax(278px, 1fr));
-    gap: var(--udexGridGutters);
-    list-style: none;
-    margin: 0;
-    padding: 0;
-  }
-
-  &.text-only {
```

Comment:
Why we need to move all styles? Can't we just move text-only styles to common CSS.